draft-ietf-scitt-architecture.md

```diff
@@ -118,6 +118,7 @@ informative:
   SWID:
     target: https://csrc.nist.gov/Projects/Software-Identification-SWID/guidelines
     title:  SWID Specification
+  EQUIVOCATION: DOI.10.1145/1323293.1294280
```

Comment:
```suggestion
  NON-EQUIVOCATION: DOI.10.1145/1323293.1294280
```

Reply:
Re my comment above in https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/131#discussion_r1407117520, this paper is cited as the first definition of equivocation, and never formally defines non-equivocation in its context, but describes it a lot.

Reply:
I left this as-is with my original citation as it defines equivocation in that article, not non-equivocation (we get that from here and other places, they just don't actually define the opposite formally with that term). If you want to push back and change the citation itself or we think it is important to mark the citation as non-equivocation through the reference in the document, reopen this and let me know.